### DISCUSSION TITLE
DMR Database Size vs Radio Model

### DISCUSSION THREAD

**Discussion by dmkacz** [hmatuschek/qdmr#498] [General]
How is the DMR database that's downloaded truncated to fit in models with varying amounts of memory? I have a 878UVii+ with 500K contact capacity and another UVii that only has 200K. Does the model recognition feature compensate, and size the download, or is the file just loaded until memory fills, and it stops at that point? I haven't tried the function on the 200K radio yet, but of course it works fine on the UVii+. What should I expect on the UVii?  If it is sized, how does the software determine what stays, and what goes in the database?

Thanks

**Comment by allesand** [comment]
QDMR sorts IDs "closest" to yours. So chances are that often needed contacts are in the radio, while those far away (ID wise, which most of the times also means in distance) are not.
Yes, with a 200k Anytone that works just fine and tuning in to TG91 you can check the result..
Details here: https://dm3mat.darc.de/qdmr/manual/ch02s11.html in "The settings dialog: Programming"
